VideoLocker/src/main/java/org/edx/mobile/discussion/DiscussionTextUtils.java

```diff
@@ -32,15 +32,17 @@ private DiscussionTextUtils() {
 
     public static void setAuthorAttributionText(@NonNull TextView textView,
                                                 @StringRes final int stringRes,
```

Comment:
I guess we could create an enum for the different string types, with methods to get the string resource with our without the label, and just take the enum as a parameter.